### ISSUE TITLE
lineWrappingIndentation does not enforce wrapping indentation in while()

### ISSUE THREAD

**Issue by vrozov** [checkstyle/checkstyle#2477]
Expected 4 errors (2 additional for while, the same as if), not 2:
```
java -jar ~/Downloads/checkstyle-6.11.2-all.jar -c 1.xml sample.java
Starting audit...
sample.java:14: error: 'b' have incorrect indentation level 20, expected level should be 6.
sample.java:15: error: 'c' have incorrect indentation level 17, expected level should be 6.
Audit done.
Checkstyle ends with 2 errors.

cat -n 1.xml
     1	<?xml version="1.0"?>
     2	<!DOCTYPE module PUBLIC
     3	  "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
     4	  "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
     5	
     6	<module name="Checker">
     7	  <property name="charset" value="UTF-8"/>
     8	
     9	  <module name="TreeWalker">
    10	    <module name="Indentation">
    11	      <property name="basicOffset" value="2"/>
    12	      <property name="braceAdjustment" value="0"/>
    13	      <property name="caseIndent" value="2"/>
    14	      <property name="throwsIndent" value="2"/>
    15	      <property name="arrayInitIndent" value="2"/>
    16	      <property name="lineWrappingIndentation" value="2"/>
    17	      <property name="forceStrictCondition" value="true"/>
    18	    </module>
    19	  </module>
    20	
    21	
    22	</module>

cat -n sample.java 
     1	public class sample {
     2	  public static void indent() {
     3	    boolean a = false;
     4	    boolean b = false;
     5	    boolean c = false;
     6	
     7	    while (a ||
     8	                    b ||
     9	                 c) {
    10	      a = true;
    11	    }
    12	
    13	    if (a ||
    14	                    b ||
    15	                 c) {
    16	      a = true;
    17	    }
    18	  }
    19	}
    20	
```

**Comment by gustahrodrigues** [comment]
It's not working for multiline string too on `Enum`. Any news about it?

```shell
$ java -jar checkstyle-8.35-all.jar -c 1.xml Sample.java
Starting audit...
Audit done.

$ cat -n 1.xml
     1  <?xml version="1.0" encoding="UTF-8"?>
     2  <!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN" "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
     3  <module name="Checker">
     4     <property name="charset" value="UTF-8" />
     5     <module name="TreeWalker">
     6        <module name="Indentation">
     7           <property name="basicOffset" value="2" />
     8           <property name="braceAdjustment" value="0" />
     9           <property name="caseIndent" value="2" />
    10           <property name="throwsIndent" value="2" />
    11           <property name="arrayInitIndent" value="2" />
    12           <property name="lineWrappingIndentation" value="2" />
    13           <property name="forceStrictCondition" value="true" />
    14        </module>
    15     </module>
    16  </module>

$ cat -n Sample.java
     1  public enum MyEnum {
     2
     3    MY_VALUE("first line"
     4          + "second line"
     5                       + "third line");
     6
     7    private final String message;
     8
     9    MyEnum(final String message) {
    10      this.message = message;
    11    }
    12  }



```
